apps/cde-ui/src/index.html

```diff
@@ -6,6 +6,8 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&display=swap" rel="stylesheet" />
```

Comment:
I don't think any of the pages uses Inter as a font so we can probably remove this